tests/functional/conftest.py

```diff
@@ -51,8 +51,14 @@ def keeper(accounts):
 
 
 @pytest.fixture
-def strategy(gov, strategist, keeper, token, vault, TestStrategy):
+def chiToken(gov, ChiToken):
+    yield gov.deploy(ChiToken)
+
+
+@pytest.fixture
+def strategy(gov, strategist, keeper, token, vault, TestStrategy, chiToken):
     strategy = strategist.deploy(TestStrategy, vault)
+    strategy.setChiToken(chiToken.address, {"from": gov})
```

Comment:
Remove `strategy.setChiToken()` from here and add another test with it activated to show that it works with or without this feature added

Reply:
> One suggested change, and then it would be nice if you could test this on Ropsten or another testnet and point to that txn hash showing it integrates properly.
> 
> If it's only on mainnet, we can try it ourselves.

So you have a deployment guide?